### ISSUE TITLE
Regarding the question of whether the lora fine-tuning model converges

### ISSUE THREAD

**Issue by oywenjun11** [stepfun-ai/Step1X-Edit#93]
I am trying to fine-tune the model to adapt to the downstream add task. During the training process, I found that the loss function has not changed significantly. Is this reasonable?
The overall loss range finally stabilized at around 0.21~0.20
![Image](https://github.com/user-attachments/assets/958f83a5-0b4b-4840-92a3-f3ca1de52525)

![Image](https://github.com/user-attachments/assets/e90043f3-0348-4779-ab52-5f76e6157325)

![Image](https://github.com/user-attachments/assets/61c568b3-c7c7-4707-b546-c566c24e6483)

![Image](https://github.com/user-attachments/assets/489f0483-2c51-439b-be88-8464f29b740a)

![Image](https://github.com/user-attachments/assets/43cb40cd-b21b-43f2-8867-51b259f9bcff)

Then when I used my lora for inference, I found that the overall image was good, but the details were not well reflected, such as the texture of the ship, the fence on the ship, etc. The following is an example:
ref_image
![Image](https://github.com/user-attachments/assets/504f1418-4cef-4c1b-8240-4c3475a88459)
caption:Add a medium-sized boat , casting a faint shadow on the surface.
![Image](https://github.com/user-attachments/assets/9afb702c-c987-460c-b6eb-82f93e0e04b8)

I wonder if it's because my training data is not good enough. Due to various factors, I only have about 100 data points, and the scene states are also different. Maybe the instructions are not detailed enough to cause the above problems.
Thanks in advance for your answers!
